blurring_as_a_service/training_pipeline/components/train_model.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
why is this file pushed again? are there differences with the training pipeline PR?

Reply:
same for the whole training_pipeline folder actually :D